fiftyone/utils/coco.py

```diff
@@ -597,7 +603,7 @@ def setup(self):
             license_map = None
 
         self._info = info
-        self._classes = classes
+        self._classes = classes_map
```

Comment:
could rename this to `self._classes_map` for consistency with other naming conventions (renaming private instance variables is not a concern from a backwards-compatibility standpoint)

Reply:
yes, absolutely!

Reply:
done